azure/templates/run-tests.yml

```diff
@@ -40,12 +50,17 @@ steps:
     - ${{ if not(parameters.full) }}:
           # In order to run tests in prod you must supply the unique ID of an Apigee app
           # that has authorized access to your service proxy.
+          - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
```

Comment:
Can remove this change, you've already loaded the secret between lines 21 and 24.